services-js/registry-certs/client/birth/components/ApostilleRequestInstructions.tsx

```diff
@@ -1,6 +1,9 @@
+/** @jsx jsx */
+
 import React, { useEffect, useState } from 'react';
 
 import { css } from 'emotion';
```

Comment:
Couldn’t this be

```suggestion
import { css, jsx } from '@emotion/core';
```
instead of separate lines?

Reply:
Not right now. `emotion`’s `css` is the legacy mode that generates a class name. `@emotion/core`’s `css` makes a style object.

I’m leaving these other apps in legacy mode for now.

Reply:
Ah, got it! We can update as we go.